test/support/websockets_helpers.rb

```diff
@@ -0,0 +1,5 @@
+module WebsocketsHelpers
+  def wait_for_websockets
+    sleep(0.5)
```

Comment:
In some future iteration, it's probably less brittle to listen for the connection itself!

Reply:
@SleeplessByte How? :)